packages/components/src/dimension-control/stories/index.story.tsx

```diff
@@ -34,7 +34,8 @@ export default {
 		controls: { expanded: true },
 		docs: { canvas: { sourceState: 'shown' } },
 	},
-} as Meta< typeof DimensionControl >;
```

Comment:
Converting this back to our standard format, because the type casting overly complicates the AST parsing we'd have to do. Not worth it right now.